### PR TITLE
Add docsify detection

### PR DESCRIPTION
**Pull request by QingWei-Li**
Hi there,

I am a creator of [docsify](//github.com/qingwei-li/docsify), It is a document generation tool like Gitbook. Currently many open source projects are using docsify to develop document site, here are some [examples](https://github.com/QingWei-Li/docsify#showcase).

I want to add it to Wappalyzer.

Thanks

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -11210,6 +11210,15 @@
 			"icon": "vwo.svg",
 			"website": "https://vwo.com/"
 
+		},
+		"docsify": {
+			"cats": [
+				"4"
+			],
+			"icon": "docsify.svg",
+			"script": "docsify",
```
Comment (jvoisin) [inline src/apps.json]: What about `/docsify/` instead?

Comment (QingWei-Li) [reply src/apps.json]: fixed

**Comment by AliasIO** [comment]
Thank you!

**Comment by QingWei-Li** [comment]
Hi,
Currently it is deleted via this commit https://github.com/AliasIO/Wappalyzer/commit/7776623fb77e7b25874695201cefddb5c0fce3f6

**Comment by jvoisin** [comment]
Wow, this was a _massive_ commit :/

**Pull request merged**